plugins/head2branch/README.md

```diff
@@ -0,0 +1,14 @@
+## Convert Head to Branch
+
+`fast-export` can only handle one head per branch. This plugin makes it possible
+to create a new branch from a head by specifying the new branch name and
+the first divergent commit for that head.  The revision number for the commit
+should be in decimal form.
+
+Note: you must run `fast-export` with `--ignore-unnamed-heads` option,
+otherwise, the conversion will fail.
+
+To use the plugin, add
```

Comment:
"... plugin, add the command line flag `--plugin head2branch=name,<rev>`. The flag can be given multiple times to name more than one head.